Assets/CodeBase/Entities/player/Player.cs

```diff
@@ -78,27 +94,70 @@ void Start()
         inputProfile.addListener(InputEvent.Key, PlayerInputProfile.moveDown, moveDown);
         inputProfile.addListener(InputEvent.Up, PlayerInputProfile.moveUp, stopClimbing);
         inputProfile.addListener(InputEvent.Up, PlayerInputProfile.moveDown, stopClimbing);
-        inputProfile.addListener(InputEvent.Key, PlayerInputProfile.jump, jump);
+        inputProfile.addListener(InputEvent.Down, PlayerInputProfile.jump, jump);
 
         //Listeners for abilities.
         inputProfile.addListener(InputEvent.Up, PlayerInputProfile.toggleIce, toggleIce);
         inputProfile.addListener(InputEvent.Up, PlayerInputProfile.toggleFire, toggleFire);
         inputProfile.addListener(InputEvent.Up, PlayerInputProfile.toggleWind, toggleWind);
         inputProfile.addListener(InputEvent.Up, PlayerInputProfile.toggleEarth, toggleEarth);
 
+        //shift key
+        inputProfile.addListener(InputEvent.Down, PlayerInputProfile.shift, shift);
```

Comment:
Dash would be a more descriptive name for the player input profile variable, though in the future it may be mapped to the jump button for a better user experience.